scss/components/_modals.scss

```diff
@@ -0,0 +1,41 @@
+// Modal
+
+.modal-overlay {
```

Comment:
Is this overriding a class set in the modal library or is there a reason why it isn't double dash like `.modal--overlay` to keep with convention?